### DISCUSSION TITLE
Make some system folder undeletable

### DISCUSSION THREAD

**Discussion by tathastu871** [termux/termux-packages#9952] [Q&A] [answered]
I regularly execute command rm -rf $HOME/.* to delete hidden files. But that also deletes the special folders like .shortcuts .termux url opener script and likewise. so i had to write all those scripts again.
Is there a way to make those scripts and folder undeletable. so rm -rf will prevent deleteing those

**Comment by thunder-coding** [comment]
Stop pinging people just for help. Have some patience! We aren't mostly paid to work for open source, and most work is volunteered.

**Comment by agnostic-apollo** [comment, marked as answer]
This isn't a termux issue.

https://lmgtfy.app/?q=linux+remove+files+from+directory+excluding+some

**Comment by ChiefMikeK** [comment]
Please ask your questions/help on reddit  
Github is not a help forum!  
## https://www.reddit.com/r/termux/
